File: app/src/main/java/com/workfort/pstuian/app/data/local/database/ColumnNames.java
package com.workfort.pstuian.app.data.local.database;

public interface ColumnNames {
    String SL = "sl";

    interface Config {
        String ID = "id";
        String ANDROID_VERSION = "android_version";
        String IOS_VERSION = "ios_version";
        String DATA_REFRESH_VERSION = "data_refresh_version";
        String API_VERSION = "api_version";
        String ADMIN_API_VERSION = "admin_api_version";
        String FORCE_REFRESH = "force_refresh";
        String FORCE_UPDATE = "force_update";
        String FORCE_REFRESH_DONE = "force_refresh_done";
        String FORCE_UPDATE_DONE = "force_update_done";
    }

    interface Slider {
        String ID = "id";
        String TITLE = "title";
        String IMAGE_URL = "image_url";
    }

    interface Faculty {
        String ID = "id";
        String SHORT_TITLE = "short_title";
        String TITLE = "title";
    }

    interface Student {
        String NAME = "name";
        String ID = "id";
        String REG = "reg";
        String PHONE = "phone";
        String LINKED_IN = "linked_in";
        String FB_LINK = "fb_link";
        String BLOOD = "blood";
        String ADDRESS = "address";
        String EMAIL = "email";
        String BATCH_ID = "batch_id";
        String SESSION = "session";
        String FACULTY_ID = "faculty_id";
        String IMAGE_URL = "image_url";
        String CV_LINK = "cv_link";
        String BIO = "bio";
    }

    interface Teacher {
        String ID = "id";
        String NAME = "name";
        String DESIGNATION = "designation";
        String BIO = "bio";
        String PHONE = "phone";
        String LINKED_IN = "linked_in";
        String FB_LINK = "fb_link";
        String ADDRESS = "address";
        String EMAIL = "email";
        String DEPARTMENT = "department";
        String BLOOD = "blood";
        String FACULTY_ID = "faculty_id";
        String IMAGE_URL = "image_url";
    }

    interface Employee {
        String ID = "id";
        String NAME = "name";
        String DESIGNATION = "designation";
        String DEPARTMENT = "department";
        String PHONE = "phone";
        String ADDRESS = "address";
        String FACULTY_ID = "faculty_id";
        String IMAGE_URL = "image_url";
    }

    interface Batch {
        String ID = "id";
        String NAME = "name";
        String TITLE = "title";
        String SESSION = "session";
        String FACULTY_ID = "faculty_id";
        String TOTAL_STUDENT = "total_student";
    }

    interface Course {
        String ID = "id";
        String COURSE_CODE = "course_code";
        String COURSE_TITLE = "course_title";
        String CREDIT_HOUR = "credit_hour";
        String FACULTY_ID = "faculty_id";
        String STATUS = "status";
    }
}

File: app/src/main/java/com/workfort/pstuian/app/data/local/database/ColumnNames.kt
package com.workfort.pstuian.app.data.local.database

object ColumnNames {
    const val SL = "sl"

    object Config {
        const val ID = "id"
        const val ANDROID_VERSION = "android_version"
        const val IOS_VERSION = "ios_version"
        const val DATA_REFRESH_VERSION = "data_refresh_version"
        const val API_VERSION = "api_version"
        const val ADMIN_API_VERSION = "admin_api_version"
        const val FORCE_REFRESH = "force_refresh"
        const val FORCE_UPDATE = "force_update"
        const val FORCE_REFRESH_DONE = "force_refresh_done"
        const val FORCE_UPDATE_DONE = "force_update_done"
    }

    object Slider {
        const val ID = "id"
        const val TITLE = "title"
        const val IMAGE_URL = "image_url"
    }

    object Faculty {
        const val ID = "id"
        const val SHORT_TITLE = "short_title"
        const val TITLE = "title"
    }

    object Student {
        const val NAME = "name"
        const val ID = "id"
        const val REG = "reg"
        const val PHONE = "phone"
        const val LINKED_IN = "linked_in"
        const val FB_LINK = "fb_link"
        const val BLOOD = "blood"
        const val ADDRESS = "address"
        const val EMAIL = "email"
        const val BATCH_ID = "batch_id"
        const val SESSION = "session"
        const val FACULTY_ID = "faculty_id"
        const val IMAGE_URL = "image_url"
        const val CV_LINK = "cv_link"
        const val BIO = "bio"
    }

    object Teacher {
        const val ID = "id"
        const val NAME = "name"
        const val DESIGNATION = "designation"
        const val BIO = "bio"
        const val PHONE = "phone"
        const val LINKED_IN = "linked_in"
        const val FB_LINK = "fb_link"
        const val ADDRESS = "address"
        const val EMAIL = "email"
        const val DEPARTMENT = "department"
        const val BLOOD = "blood"
        const val FACULTY_ID = "faculty_id"
        const val IMAGE_URL = "image_url"
    }

    object Employee {
        const val ID = "id"
        const val NAME = "name"
        const val DESIGNATION = "designation"
        const val DEPARTMENT = "department"
        const val PHONE = "phone"
        const val ADDRESS = "address"
        const val FACULTY_ID = "faculty_id"
        const val IMAGE_URL = "image_url"
    }

    object Batch {
        const val ID = "id"
        const val NAME = "name"
        const val TITLE = "title"
        const val SESSION = "session"
        const val FACULTY_ID = "faculty_id"
        const val TOTAL_STUDENT = "total_student"
    }

    object Course {
        const val ID = "id"
        const val COURSE_CODE = "course_code"
        const val COURSE_TITLE = "course_title"
        const val CREDIT_HOUR = "credit_hour"
        const val FACULTY_ID = "faculty_id"
        const val STATUS = "status"
    }
}